modules/oe_theme_content_event/src/Plugin/ExtraField/Display/EventExtraFieldBase.php

```diff
@@ -53,4 +54,25 @@ public static function create(ContainerInterface $container, array $configuratio
     );
   }
 
+  /**
+   * Gets rendered address field from Venue entity.
+   *
+   * @param \Drupal\oe_content_entity_venue\Entity\VenueInterface $venue
+   *   Venue entity.
+   *
+   * @return string
+   *   Rendered address field.
+   */
+  public function getVenueInlineAddress(VenueInterface $venue): string {
+    $location = [];
+    $renderable = $this->entityTypeManager->getViewBuilder('oe_venue')->viewField($venue->get('oe_address'));
+    foreach (['locality', 'country'] as $field) {
+      if (!empty($renderable[0][$field]['#value'])) {
+        $location[] = $renderable[0][$field]['#value'];
+      }
+    }
+
+    return implode(', ', $location);
```

Comment:
I know it was like this before but like this we may lose cacheability, so let's render the field before, like we do here: https://github.com/ec-europa/ewcms/pull/1015/files#diff-0a7bee81a5ae430d8258164da937320de3cf918fd4c77ffaff231215d8294f26R463

Reply:
Done, but refactoring of Event's extra fields were required for it.